README.md

```diff
@@ -157,36 +157,36 @@ GIF compression is not implemented yet. For now we recommend [https://gifcompres
 
 # Usage
 
-> You can use the `--help` option to get more details about the commands and their options
+> You can use the `--help` option to get more details about the commands and their options:
```

Comment:
This should be a period instead of the colon as the text shown is the usage, shouldn't it?

```bash
terminalizer <command> [options]
```